src/content/docs/logs/ui-data/timestamp-support.mdx

```diff
@@ -0,0 +1,325 @@
+---
+title: Understanding and using timestamps in logs
```

Comment:
Since this is reference material, it should be OK to follow the ui-data naming conventions. I think 'Timestamp formats in logs' is good enough